state/txindex/indexer_service.go

```diff
@@ -33,7 +33,11 @@ func NewIndexerService(
 	terminateOnError bool,
 ) *IndexerService {
 
-	is := &IndexerService{txIdxr: txIdxr, blockIdxr: blockIdxr, eventBus: eventBus, terminateOnError: terminateOnError}
+	is := &IndexerService{
+		txIdxr:           txIdxr,
+		blockIdxr:        blockIdxr,
+		eventBus:         eventBus,
+		terminateOnError: terminateOnError}
```

Comment:
Do you use a formatter locally in your IDE, like `gofumpt`?

Reply:
fixed